microcosm_flask/routing.py

```diff
@@ -40,6 +41,9 @@ def search_foo():
     """
     # routes depends on converters
     graph.use(*graph.config.route.converters)
+    # routes are annotated to make request context availale in
+    # graph.opaque
+    graph.use('opaque')
```

Comment:
Consider moving this into `factories.py` instead; I only put the converters here because I was trying to support user-defined converters.